abb_irb1600_support/launch/test_irb1600_6_12.launch

```diff
@@ -0,0 +1,7 @@
+<launch>
+        <include file="$(find abb_irb1600_support)/launch/load_irb1600_6_12.launch" />
+	<param name="use_gui" value="true" />
+	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
+	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
+	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find industrial_robot_client)/config/robot_state_visualize.rviz" required="true" />
+</launch>
```

Comment:
And would be nice if `use_gui` could be made a private param of `joint_state_publisher`.